packages/app/src/systems/Transaction/components/TxRecipientCard/TxRecipientCard.tsx

```diff
@@ -0,0 +1,80 @@
+import { cssObj } from "@fuel-ui/css";
+import { Avatar, Box, Card, Copyable, Flex, Icon, Text } from "@fuel-ui/react";
+import type { FC } from "react";
+
+import { TxRecipientCardLoader } from "./TxRecipientCardLoader";
+
+import type { Account } from "~/systems/Account";
+import { shortAddress } from "~/systems/Core";
+
+export type TxRecipientCardProps = {
+  account?: Account;
+  contract?: { address: string };
+};
+
+type TxRecipientCardComponent = FC<TxRecipientCardProps> & {
+  Loader: typeof TxRecipientCardLoader;
+};
+
+export const TxRecipientCard: TxRecipientCardComponent = ({
+  account,
+  contract,
+}) => {
+  const address = account ? account.address : contract?.address;
+  return (
+    <Card css={styles.root}>
+      <Text css={styles.from}>From</Text>
```

Comment:
can we make From font size a little bit smaller? figma mockup is little bit smaller and feels better to me

Reply:
✅